requirements.txt

```diff
@@ -5,3 +5,4 @@ cpsk
 pygeocoder
 brutal-ng==0.3.9
 imhdsk
+calendar_sk
```

Comment:
@mrshu Hmmm, I would rather use the latest version of calendar and not to specify it each time I make some small change.